### PR TITLE
feat(RadioMe): Complete recode

### PR DESCRIPTION
**Pull request by ACertainCoder**
**Describe the changes in this pull request:**
Completely recoded the RadioMe presence to work with the current website structure.

<details>
<summary> Proof to proving the creation/modification working </summary>
<br>
RadioMe

</details>

### CODE REVIEW

File: websites/R/RadioMe/presence.ts [hunk 1]
```diff
@@ -4,217 +4,124 @@ const presence = new Presence({
   strings = presence.getStrings({
     play: "presence.playback.playing",
     pause: "presence.playback.paused",
-    browse: "presence.activity.browsing",
-    search: "presence.activity.searching"
+    search: "presence.activity.searching",
+    browsing: "presence.activity.browsing"
```
Comment (Bas950) [inline websites/R/RadioMe/presence.ts:8]: use general. strings.

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
- Why did you decide to recode this? 
- Why don't you use a Record instead of the extensive switch-case statement?
- How is the multiLang change handled?

Please remove the exessive:
- Obvious comments, keep them where appropriate, not everywhere 
- `presence.setActivity()`s

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/R/RadioMe/presence.ts [hunk 2]
```diff
@@ -1,220 +1,134 @@
...
+ * @param path Page path
+ */
+async function handleUnknown(path: string[]): Promise<void> {
+  const region = [
+    ...document.querySelectorAll<HTMLAnchorElement>(".region-btn")
+  ]
+    .filter(e => e.classList.contains("active"))[0]
+    .pathname?.slice(1);
+
+  if (path[0] && region !== "" && path[0] === region) {
```
[the remark this answers is not in the document]
Comment (ACertainCoder) [reply websites/R/RadioMe/presence.ts:122]: grafik
The region can be an empty string

[the remark this answers is not in the document]
Comment (ImRodry) [reply websites/R/RadioMe/presence.ts:122]: Yeah and an empty string is falsy

File: websites/R/RadioMe/presence.ts [hunk 3]
```diff
@@ -1,220 +1,131 @@
...
+function handleStation(): void {
+  const station =
+    document.querySelector<HTMLSpanElement>(".song-name")?.textContent;
+
+  let presenceData: PresenceData;
+
+  // Check if the playing icon is shown
+  if (document.querySelector<HTMLDivElement>(".playbutton-global-playing")) {
+    // Radio is playing / buffering
+    timestamp &&= Date.now();
```
Comment (ImRodry) [inline websites/R/RadioMe/presence.ts:52]: Nope that’s not what you wanna do
```suggestion
    timestamp ||= Date.now();
```

File: websites/R/RadioMe/presence.ts [hunk 4]
```diff
@@ -1,220 +1,131 @@
...
+ * @param path Page path
+ */
+function handleUnknown(path: string[]): void {
+  const region = [
+    ...document.querySelectorAll<HTMLAnchorElement>(".region-btn")
+  ]
+    .filter(e => e.classList.contains("active"))[0]
+    .pathname?.slice(1);
+
+  if (region !== "" && path[0] === region) {
```
Comment (ImRodry) [inline websites/R/RadioMe/presence.ts:122]: You don’t get it do ya
```suggestion
  if (region && path[0] === region) {
```

File: websites/R/RadioMe/presence.ts (hunk 3 above)
Comment (ACertainCoder) [reply websites/R/RadioMe/presence.ts:52]: Well, they linked me to the logical AND page. ¯\_(ツ)_/¯
I'm going to change it.

File: websites/R/RadioMe/presence.ts (hunk 4 above)
Comment (ACertainCoder) [reply websites/R/RadioMe/presence.ts:122]: An example would have been useful the first time...
I'm going to change it.

Comment (ImRodry) [reply websites/R/RadioMe/presence.ts]: You should know that empty strings are falsy

Comment (ACertainCoder) [reply websites/R/RadioMe/presence.ts]: Sure but I didn't see what you ment directly. Sorry but after more than 8 hours working on these presences explaining various decisions it's hard to calm down and think straight.

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**